src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const res = { };
```

Comment:
```suggestion
  const res = {};
```

I think, that this inner space is unnecessary in empty object declaration, do you?